bug: could not install

## Description

could not install

## Package version
latest

## Error Message & Stack Trace
```node
npm ERR! path /Users/Username/.nvm/versions/node/v12.6.0/lib/node_modules/gh-copy-labels/build/bin.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/Username/.nvm/versions/node/v12.6.0/lib/node_modules/gh-copy-labels/build/bin.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Username/.npm/_logs/2019-07-29T10_58_09_742Z-debug.log
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,6 +1,11 @@
 {
   "extends": "./node_modules/@adonisjs/mrm-preset/_tsconfig",
+  "include": [
+    "src/**/*"
+  ],
   "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build",
     "lib": [
       "es2015",
       "es2016"

